hs/Css/Timetable.hs

```diff
@@ -178,13 +178,11 @@ courseSelectCSS = do
         margin0
         padding0
         height100
-        overflow hidden
         backgroundColor purple8
         color white
     "#course-select" ? do
-        padding0
-        margin (px 18) 0 0 (px 17)
-        width100
+        padding (px 18) 0 0 (px 17)
+        margin0
         height100
         alignCenter
         overflowY scroll
```

Comment:
Does it always add to the width of the div? I thought it subtracted from the available width of the div contents.